yarn.lock

```diff
@@ -6047,8 +6047,8 @@ __metadata:
   linkType: hard
 
 "protobufjs@npm:^7.2.2":
-  version: 7.2.2
-  resolution: "protobufjs@npm:7.2.2"
+  version: 7.4.0
```

Comment:
Doesn't match the description, but this might be fine? Here are the additional releases:
https://github.com/protobufjs/protobuf.js/releases?q=protobufjs&expanded=true

Reply:
That's an odd mismatch